cmd/srcd/cmd/web_test.go

```diff
@@ -0,0 +1,158 @@
+// +build integration
+
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/blang/semver"
+	cmdtest "github.com/src-d/engine/cmd/test-utils"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type WebTestSuite struct {
+	cmdtest.IntegrationSuite
+	testDir string
+}
+
+func TestWebTestSuite(t *testing.T) {
+	s := WebTestSuite{}
+	suite.Run(t, &s)
+}
+
+func (s *WebTestSuite) testCommon(subcmd string, assertions func(url string)) {
+	require := s.Require()
+
+	var out bytes.Buffer
+
+	command := s.CommandContext(context.TODO(), "web", subcmd)
+	command.Stdout = &out
+	command.Stderr = &out
```

Comment:
why not writing `stderr` into a new error buffer, and then assert that if there were no errors, there is nothing written in the error buffer?

Reply:
We are sending the log messages to `stderr`. We could simply ignore `stderr` and capture `stdout` only, but I left it combined to work in a consistent way with `RunCommand`.

Reply:
errors in `stdout` :(
ok...
given that constraint, consistency was a good decision :+1:

Reply:
No, error messages also go to `stderr`.